cloudformation/membership-attribute-service.yaml

```diff
@@ -75,6 +58,9 @@ Mappings:
       DynamoDBTableTestUsers: "Memb-Attributes-Tables-UAT-SupporterAttributesFallback"
       DynamoDBFeatureToggleTable: arn:aws:dynamodb:*:*:table/MembershipFeatureToggles-PROD
       DynamoDBFeatureToggleTableTestUsers: arn:aws:dynamodb:*:*:table/MembershipFeatureToggles-UAT
+      ReadableS3Resources:
+        - arn:aws:s3:::gu-membership-attribute-service-dist/PROD/*
```

Comment:
this should probably be arn:aws:s3:::gu-membership-attribute-service-dist/membership/PROD/*

Reply:
Thanks - I will re-test this on CODE before I release this